huaweicloud/services/coc/resource_huaweicloud_coc_incident.go

```diff
@@ -0,0 +1,371 @@
+package coc
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+var incidentNonUpdatableParams = []string{"region", "enterprise_project", "start_time", "current_cloud_service",
+	"incident_level", "is_service_interrupt", "incident_type", "incident_ownership", "incident_title",
+	"incident_description", "incident_source", "assignee_scene", "assignee_role", "creator"}
```

Comment:
check the params name
```suggestion
var incidentNonUpdatableParams = []string{"regions", "enterprise_project", "start_time", "current_cloud_service",
	"incident_level", "is_service_interrupt", "incident_type", "incident_ownership", "incident_title",
	"incident_description", "incident_source", "assignee_scene", "assignee_role", "creator"}
```